### ISSUE TITLE
OSVR RenderManager Could not acquire display access for vendor ID 0xd24e for display 0

### ISSUE THREAD

**Issue by dtmoodie** [OSVR/OSVR-Core#390]
Hi,

Windows 10x64
Nvidia GTX titanX x 2 - 364.51
OSVR runtime v0.6-1121-gf4ea7ef-win-64bit
OSVR sdk - v0.6-1121-gf4ea7ef-win-64bit
Render manager - 0_6_41_unsigned

OSVR server log:
----------------------------------------------------------------------------------------------------------------------
[OSVR Server] Using default config file - pass a filename on the command line to use a different one.
[OSVR Server] Using config file 'osvr_server_config.json'
[OSVR Server] Constructing server as configured...
[OSVR Server] Loading auto-loadable plugins...
[OSVR Server] Loading plugins...
[OSVR Server]
[OSVR Server] Instantiating configured drivers...
[OSVR] Added device: org_osvr_filter_videoimufusion/HeadFusion
[OSVR] Client context initialized for org.osvr.analysisplugin
[OSVR] Interface initialized for /com_osvr_Multiserver/OSVRHackerDevKitPrediction0/semantic/hmd
[OSVR] Interface initialized for /com_osvr_VideoBasedHMDTracker/TrackedCamera0_0/semantic/hmd/front
[OSVR Server] Successes:
[OSVR Server]  - com_osvr_VideoBasedHMDTracker/VideoBasedHMDTracker
[OSVR Server]  - org_osvr_filter_videoimufusion/VideoIMUFusion
[OSVR Server]
[OSVR Server]
[OSVR Server] Aliases found and parsed from config file.
[OSVR Server] Display descriptor found and parsed from config file.
[OSVR Server] RenderManager config found and parsed from the config file.
[OSVR Server] Triggering automatic hardware detection...
[OSVR Server] Registering shutdown handler...
[OSVR Server] Starting server mainloop: OSVR Server is ready to go!
[OSVR] Performing hardware auto-detection.
[OSVR] Added device: com_osvr_Multiserver/OSVRHackerDevKit0
[OSVR] Added device: com_osvr_Multiserver/OSVRHackerDevKitPrediction0
directx_camera_server: Using capture device 'USB Camera' at path '\\?\usb#vid_0bda&pid_57e8&mi_00#7&314fc184&0&0000#{65e8773d-8f56-11d0-a3b9-00a0c9223196}\global'
Video-based tracker: Camera turned on!
Opening camera 0
[OSVR] Added device: com_osvr_VideoBasedHMDTracker/TrackedCamera0_0
Video-based tracker: NOTE: Beacon calibration filename videotrackerCombinedCalibrationFile13.json was specified, but not found or could not be loaded. This is not an error: This may just mean you have not yet run the optional beacon pre-calibration step.
[OSVR] Path tree updated or connection detected
[OSVR] Sending path tree to clients.
[OSVR] Got updated path tree, processing
[OSVR] Constructed a TrackerHandler for com_osvr_Multiserver/OSVRHackerDevKitPrediction0@localhost sensor 0
[OSVR] Successfully produced handler for /com_osvr_Multiserver/OSVRHackerDevKitPrediction0/semantic/hmd
[OSVR] Constructed a TrackerHandler for com_osvr_VideoBasedHMDTracker/TrackedCamera0_0@localhost sensor 0
[OSVR] Successfully produced handler for /com_osvr_VideoBasedHMDTracker/TrackedCamera0_0/semantic/hmd/front
[OSVR] Connected 2 of 2 unconnected paths successfully
[OSVR] Performing hardware auto-detection.
[OSVR] Path tree updated or connection detected
[OSVR] Sending path tree to clients.
vrpn: Connection request received from 127.0.0.1: 127.0.0.1 2690
[OSVR] Got first client connection, exiting idle mode.
[OSVR] Performing hardware auto-detection.
[OSVR] Path tree updated or connection detected
[OSVR] Sending path tree to clients.
vrpn: Connection request received from 127.0.0.1: 127.0.0.1 2692
[OSVR] Performing hardware auto-detection.
[OSVR] Path tree updated or connection detected
[OSVR] Sending path tree to clients.
vrpn_Endpoint::getOneTCPMessage:  Can't read header (this is normal when a connection is dropped)
vrpn: TCP handling failed, dropping connection (this is normal when a connection is dropped)
[OSVR] Performing hardware auto-detection.
[OSVR] Path tree updated or connection detected
[OSVR] Sending path tree to clients.
[OSVR] Performing hardware auto-detection.
[OSVR] Path tree updated or connection detected
[OSVR] Sending path tree to clients.
vrpn_Endpoint::send_pending_reports:  TCP send failed.
vrpn_Connection_IP::send_pending_reports:  Closing failed endpoint.
[OSVR] Dropped last client connection, entering idle mode.
[OSVR] Performing hardware auto-detection.
[OSVR] Path tree updated or connection detected
[OSVR] Sending path tree to clients.

----------------------------------------------------------------------------------------------------------------
RenderManagerOpenGLExample.exe
[OSVR] Connecting to default (local) host
[OSVR] Client context initialized for com.osvr.renderManager.openGLExample
[OSVR] Got connection to main OSVR server
[OSVR] Got updated path tree, processing
[OSVR] Connected 0 of 0 unconnected paths successfully
[OSVR] Connection process took 106ms: have connection to server, have path tree
[OSVR] Interface initialized for /controller/left/1
[OSVR] Could not resolve source for /controller/left/1
[OSVR] Interface initialized for /controller/right/1
[OSVR] Could not resolve source for /controller/right/1
[OSVR] Connecting to default (local) host
[OSVR] Client context initialized for com.osvr.renderManager
[OSVR] Got connection to main OSVR server
[OSVR] Got updated path tree, processing
[OSVR] Connected 0 of 0 unconnected paths successfully
[OSVR] Connection process took 102ms: have connection to server, have path tree
[OSVR] Interface initialized for /me/head
[OSVR] Constructed a TrackerHandler for org_osvr_filter_videoimufusion/HeadFusion@localhost sensor 0
[OSVR] Successfully produced handler for /me/head
[OSVR] Interface initialized for /me/head
[OSVR] Interface initialized for /me/head
[OSVR] Display: OSVR HDK (Version 1.3) [Display descriptor note: Specific to the optics of 1.3, with Render Manager compatible distortion parameters]
[OSVR] Created an OSVR_DisplayConfigObject!
RenderManager::createRenderManager(): Got Display info from server (ignore earlier errors that occured while we were waiting to connect)
OSVRDisplayConfiguration::parse(): Using polynomial distortion.
[OSVR] Interface initialized for /me/head
[OSVR] Interface initialized for /me/head
[OSVR] Interface initialized for /me/head
[OSVR] Interface initialized for /me/hands/left
[OSVR] Could not resolve source for /me/hands/left
[OSVR] Interface initialized for /me/hands/right
[OSVR] Could not resolve source for /me/hands/right
OSVR RenderManager Warning: Got error from NVIDIA API: NVAPI call returned -208 (hex: ffffff30) [NVAPI_HDCP_DISABLED]
RenderManagerNVidiaD3D11::OpenDisplay: Could not acquire display access for vendor ID 0xd24e for display 0
RenderManagerD3D11OpenGL::OpenDisplay: Can't open D3D11 display
Could not open display
[OSVR] Client context shut down for com.osvr.renderManager.openGLExample
vrpn_Endpoint::send_pending_reports():  select() failed.
Error (10093):  Either the application has not called WSAStartup, or WSAStartup failed.
.
vrpn_Connection_IP::send_pending_reports:  Closing failed endpoint.
vrpn_Connection::~vrpn_Connection: Connection was deleted while 6 references still remain.

**Comment by rpavlik** [comment]
This looks like an issue we've noted with the 364.51 NVIDIA drivers. You need to downgrade to 362.00 or 361.92.

**Comment by russell-taylor** [comment]
Yes, I've talked with nVidia about this and am working on a patch that will work with their drivers going forward.  This is a new feature that reports when you are unable to display copyright content.  We need to treat it as not an error.  Previously, anything but NVAPI_OK was an error; this is more of a warning.

**Comment by russell-taylor** [comment]
The initial workaround suggested by nVidia didn't solve the problem, so I've asked for a new API version that we can use.

**Comment by rpavlik** [comment]
We're apparently not the only ones that got surprised by this one: http://store.steampowered.com/news/externalpost/steam_community_announcements/295355928885907590
